mentors/scala/b.json

```
@@ -1 +1,10 @@
-[]
+[
+{
```

```suggestion
  {
```
indent two spaces.  Lines that follow `{` should also be indented two spaces in from `[`.